pymic/offload_stream.py

```diff
@@ -562,6 +562,9 @@ def invoke(self, kernel, *args):
             raise OffloadError("Cannot invoke kernel, "
                                "library not loaded on device")
 
+        # Map None to nullptr
+        args = [arg if arg is not None else 0 for arg in args]
```

Comment:
The interface of the kernels will very likely stay as they are.  I’m ok with the current patch for now to buy some more time for the LIBXSTREAM developers.
Intel Deutschland GmbH
Registered Address: Am Campeon 10-12, 85579 Neubiberg, Germany
Tel: +49 89 99 8853-0, www.intel.de
Managing Directors: Christin Eisenschmid, Prof. Dr. Hermann Eul
Chairperson of the Supervisory Board: Tiffany Doon Silva
Registered Office: Munich
Commercial Register: Amtsgericht Muenchen HRB 186928